src/components/header/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Regle from './components/Regle';
+import logo from './components/assets/logo.png';
+
+function Header() {
+  return (
+    <>
+      <img className="logo" src={logo} alt="Age of Memory" />
+      <Regle />
```

Comment:
names should be in english :)

change it for Rules, and push the commit in the same branch